github/commitstatus.go

```diff
@@ -175,9 +183,16 @@ func httpRequestDo(req *http.Request) (httpResponse, error) {
 	response.oauthInfo = fmt.Sprintf("X-Accepted-Oauth-Scopes: %v, X-Oauth-Scopes: %v",
 		resp.Header.Get("X-Accepted-Oauth-Scopes"), resp.Header.Get("X-Oauth-Scopes"))
 
-	response.rateLimitRemaining, _ = strconv.Atoi(resp.Header.Get("x-ratelimit-remaining"))
-	rateLimitReset, _ := strconv.Atoi(resp.Header.Get("x-ratelimit-reset"))
+	response.rateLimitRemaining, _ = strconv.Atoi(resp.Header.Get("X-RateLimit-Remaining"))
+	rateLimitReset, _ := strconv.Atoi(resp.Header.Get("X-RateLimit-Reset"))
 	response.rateLimitReset = time.Unix(int64(rateLimitReset), 0)
+
+	// time.RFC1123 is the format of the HTTP Date header
+	// example: Date:[ Mon, 02 Jan 2006 15:04:05 MST ]
+	// https://datatracker.ietf.org/doc/html/rfc2616#section-14.18
+	date, _ := time.Parse(time.RFC1123, resp.Header.Get("Date"))
```

Comment:
same comment should apply here also: https://github.com/Pix4D/fly_helper/pull/158/files#r1201798389

> [marco-m-pix4d](https://github.com/marco-m-pix4d)(Marco Molteni) [2 minutes ago](https://github.com/Pix4D/fly_helper/pull/158/files#r1201798389)
In Go, never skip handling errors, unless there is a very specific reason.
This is equivalent to intercepting all exceptions in Python and dropping them.